app/models/delayed_job.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+
+class DelayedJob < Delayed::Job; end
```

Comment:
This is required as administrate works best with non-namespaced models. (It is possible to create dashboard for namespaced models, however, as UI text and URLs are generated based on the model class, this is a little nicer, I think.)